src/components/slider/slider.stories.ts

```diff
@@ -279,3 +279,17 @@ export const wordBreakDoesNotAffectLabels_TestOnly = (): string =>
   ></calcite-slider>`;
 
 export const WithLabelHandlesAndNoValue_TestOnly = (): string => html` <calcite-slider max="750"></calcite-slider> `;
+
+export const WithLargeFontSize_TestOnly = (): string => html`<html lang="en">
+  <head>
+    <style>
+      html {
+        font-size: 20px;
+      }
+    </style>
+  </head>
+  <body>
+    <calcite-slider label-handles min-value="50" max-value="70" range></calcite-slider>
+    <calcite-slider label-handles value="70"></calcite-slider>
```

Comment:
should we add all types of slider's  available to users in this test?

Reply:
I ended up adding more slider variants in a single test. This can be hard to maintain in terms of updating the test but adds value in identifying regression for larger font-size.